### PR TITLE
WWST-1039 Add Health Check to Kickstarter motion sensor

### PR DESCRIPTION
**Pull request by greens**

### CODE REVIEW

File: devicetypes/smartthings/smartsense-motion.src/smartsense-motion.groovy [hunk 1]
```diff
@@ -43,6 +44,11 @@ metadata {
 	}
 }
 
+def installed() {
+	// device checks in every 2.5 minutes, but we'll give it the same checkinterval as our other devices
+	sendEvent(name: "checkInterval", value: 60 * 12, displayed: false, data: [protocol: "zigbee", hubHardwareId: device.hub.hardwareID, offlinePingable: "1"])
```
Comment (tpmanley) [inline devicetypes/smartthings/smartsense-motion.src/smartsense-motion.groovy]: I don't believe this device is offlinePingable.

**Comment by workingmonk** [comment]
probably better to target this to master. We 13k device installs for this DTH and need good testing before we deploy the change

**Comment by greens** [comment]
👍

**Comment by greens** [comment]
ping

**Review by tpmanley — APPROVED** [review]

**Pull request merged**